RobotLocation[] robotList
uint8 numRobots